modules/utxo-lib/src/testutil/psbt.ts

```diff
@@ -108,14 +108,16 @@ export function getSigners(inputType: InputScriptType): { signerName: KeyName; c
  * signs with first or second signature for single input.
  * p2shP2pk is signed only with first sign.
  */
-export function signPsbtInput(
-  psbt: UtxoPsbt,
-  input: Input,
-  inputIndex: number,
-  rootWalletKeys: RootWalletKeys,
-  sign: 'halfsigned' | 'fullsigned',
-  signers?: { signerName: KeyName; cosignerName?: KeyName }
-): void {
+export function signPsbtInput(params: {
+  psbt: UtxoPsbt;
+  input: Input;
+  inputIndex: number;
+  rootWalletKeys: RootWalletKeys;
+  sign: 'halfsigned' | 'fullsigned';
+  signers?: { signerName: KeyName; cosignerName?: KeyName };
+  deterministic?: boolean;
```

Comment:
it would probably be enough to make only the last two params named, no?

Reply:
Few more params are incoming in a follow up PR. So becoming harder to pass arguments.

Reply:
Done